extension/chrome/content/download.js

```diff
@@ -36,7 +36,9 @@
 
 var Cc = Components.classes;
 var Ci = Components.interfaces;
+var Cu = Components.utils;
 
+Cu.import("resource://gre/modules/Downloads.jsm");
```

Comment:
We also have to bump the max version to 38.0 and release a new version of addon.